lib/feeder/api.ex

```diff
@@ -1,58 +1,55 @@
 defmodule Feeder.Api do
+  use Memoize
 
-  @per 100
+  @per 200
```

Comment:
Feeder performs better these days (fixed some N+1 queries) ... so 200 per page is now reasonable.